python/pywatchman/bser.c

```diff
@@ -971,19 +978,21 @@ static PyObject *bser_loads(PyObject *self, PyObject *args, PyObject *kw)
   int datalen = 0;
   const char *end;
   int64_t expected_len;
+  int bser_version = 1;
   PyObject *mutable_obj = NULL;
   const char *value_encoding = NULL;
   const char *value_errors = NULL;
-  unser_ctx_t ctx = {1, 0};
+  unser_ctx_t ctx = { .mutable = 1, .value_encoding = NULL };
```

Comment:
Oh this. I'm not entirely sure what mutable does, but I think all I did was just explicitly name the struct members during initialization, since I added a new field to that struct. I'll switch to the old notation {1, 0}.